array/main.js

```diff
@@ -0,0 +1,107 @@
+const log = console.log;
+let sports = ["cricket", "football", "pool", "kabaddi"]; //declare array
+log(sports[0]); // print cricket
+log(sports[4]); // undefined
+
+
+let last = sports[sports.length - 1];
+log(last); // O/P kabaddi
+
+// looping array
+sports.forEach(function (item, index, array) {
+  log(item, index); //cricket 0 football 1 pool 2 kabaddi 3
+});
+
+// add to the end of an array
+let addSport = sports.push("kho-kho");

```

Comment:
No need to have an explicit variable here if you aren't going to use it in the file. Just use the statement `sports.push("kho-kho");`
Do it similarly for some variables below.